sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -737,12 +737,19 @@ class SparkSession private(
   // scalastyle:on
 
   /**
-   * Stop the underlying `SparkContext`.
+   * Stop the underlying `SparkContext` with default exit code 0.
    *
    * @since 2.0.0
    */
-  def stop(): Unit = {
-    sparkContext.stop()
+  def stop(): Unit = stop(0)
```

Comment:
I think it is better to not change this line. This line builds on `sparkContext.stop()`. Even though the underlying implementation is `SparkContext.stop(0)`, I think we'd better do not make that assumption that it will always be.